pkg/ocicni/ocicni.go

```diff
@@ -912,6 +912,10 @@ func buildCNIRuntimeConf(cacheDir string, podNetwork *PodNetwork, ifName string,
 		rt.CapabilityArgs["ipRanges"] = runtimeConfig.IpRanges
 	}
 
+	// Set Aliases in Capabilities
+	if len(podNetwork.Aliases) > 0 {
+		rt.CapabilityArgs["aliases"] = podNetwork.Aliases
```

Comment:
there are several ways of passing information to the CNI plugin, 
https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md
but the plugin should be able to handle it, are these supposed to work with standard plugins?
I think that @squeed will be able to provide some advice on the best way to provide this info to the CNI

Reply:
@mccv1r0 advised me to pass in this manner and it somewhat follows what is done for port mapping.